### ISSUE TITLE
DONT USE!

### ISSUE THREAD

**Issue by Awire9966** [Byrbo/NebulaSpammer-new#1]
The EXE file is a token grabber. Build this yourself with visual studio 2022
Visual Studio: https://visualstudio.com/

**Comment by GoldenretriverYT** [comment]
Yeah, probably. Thats why you always should compile stuff like that (or generally most things) yourself. Not being able to decompile with ILSpy is very odd for a C# program. The binary definitely is not a C# program.

Ok, so I checked it and nope, not a .NET executable
![image](https://user-images.githubusercontent.com/49924528/162262124-af1ed6dd-a45e-4565-9bb9-36d9f3a74de0.png)

**Comment by pizzalover32** [comment]
its in pyinstaller.

**Comment by Awire9966** [comment]
Rdimo was able to grab the webhook and then i spammed it for an hour.
